src/components/advanced-color-picker/advanced-color-picker.spec.js

```diff
@@ -0,0 +1,390 @@
+import React from 'react';
+import EnzymeToJson from 'enzyme-to-json';
```

Comment:
Might be missing something but I don't think you need this import as the snapshot serialiser is now handled in `jest.conf.json`
https://github.com/Sage/carbon/blob/1c6ed38834ed3416055d832f37a435926f5f3e45/jest.conf.json#L10

Reply:
yes, fixed